README.rst

```diff
@@ -63,14 +63,15 @@ Feature extraction:
         >>> for feature, value in zip(features, values):
         ...     print("{0}: {1}".format(feature, value))
         ...
-        <bytes_changed>: 3
-        <chars_added>: 8
-        <day_of_week_in_utc>: 6
-        <hour_of_day_in_utc>: 19
-        <is_custom_comment>: True
-        <user_age_in_seconds>: 71821407
-        <user_is_anon>: False
-        <user_is_bot>: False
+        <revision.day_of_week>: 6
+        <revision.hour_of_day>: 19
+        <revision.has_custom_comment>: True
+        <(revision.bytes - parent_revision.bytes_changed)>: 3
```

Comment:
Wouldn't it be clearer if this line were
`<(revision.bytes - parent_revision.bytes)>: 3`
(this difference is what I would call the number of "bytes changed" in a diff, and might be the reason why the subtraction in line 69 looks weird to me)